src/registry/views/static/component-info.js

```diff
@@ -1,6 +1,6 @@
 'use strict';
```

Comment:
This is client-side code that then gets uglified by jade import - when making this switch the uglification fails :(
Can we add a file ignore for this rule in this file please?